projects/plugins/jetpack/_inc/lib/class.media-summary.php

```diff
@@ -262,11 +265,13 @@ public static function get( ?int $post_id, int $blog_id = 0, array $args = array
 					++$number_of_paragraphs;
 				}
 
-				$return['image']           = $extract['image'][0]['url'];
-				$return['secure']['image'] = self::ssl_img( $return['image'] );
-				++$return['count']['image'];
+				if ( isset( $extract['image'][0]['url'] ) ) {
+					$return['image']           = $extract['image'][0]['url'];
```

Comment:
I wonder if consumers of this method always expect the `image` key to be present. For example `enhanced_og_image` [here](https://github.com/Automattic/jetpack/blob/7cd3d960bcef046053b38c07f9ca3c7fdbe0d8ea/projects/plugins/jetpack/enhanced-open-graph.php#L40) would produce `Undefined array key` Warnings